plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/s3select/S3SelectDefaultTestConfig.java

```diff
@@ -0,0 +1,233 @@
+import io.trino.hdfs.DynamicHdfsConfiguration;
+import io.trino.hdfs.HdfsConfig;
+import io.trino.hdfs.HdfsConfiguration;
+import io.trino.hdfs.HdfsConfigurationInitializer;
+import io.trino.hdfs.HdfsEnvironment;
+import io.trino.hdfs.authentication.NoHdfsAuthentication;
+import io.trino.plugin.base.CatalogName;
+import io.trino.plugin.hive.AbstractTestHiveFileSystem;
+import io.trino.plugin.hive.DefaultHiveMaterializedViewMetadataFactory;
+import io.trino.plugin.hive.GenericHiveRecordCursorProvider;
+import io.trino.plugin.hive.HiveConfig;
+import io.trino.plugin.hive.HiveLocationService;
+import io.trino.plugin.hive.HiveMetadataFactory;
+import io.trino.plugin.hive.HivePageSourceProvider;
+import io.trino.plugin.hive.HivePartitionManager;
+import io.trino.plugin.hive.HiveSplitManager;
+import io.trino.plugin.hive.HiveTransactionManager;
+import io.trino.plugin.hive.LocationService;
+import io.trino.plugin.hive.NamenodeStats;
+import io.trino.plugin.hive.NodeVersion;
+import io.trino.plugin.hive.NoneHiveRedirectionsProvider;
+import io.trino.plugin.hive.PartitionUpdate;
+import io.trino.plugin.hive.PartitionsSystemTableProvider;
+import io.trino.plugin.hive.PropertiesSystemTableProvider;
+import io.trino.plugin.hive.aws.athena.PartitionProjectionService;
+import io.trino.plugin.hive.fs.FileSystemDirectoryLister;
+import io.trino.plugin.hive.metastore.HiveMetastoreConfig;
+import io.trino.plugin.hive.metastore.HiveMetastoreFactory;
+import io.trino.plugin.hive.metastore.thrift.BridgingHiveMetastore;
+import io.trino.plugin.hive.s3.HiveS3Config;
+import io.trino.plugin.hive.s3.TrinoS3ConfigurationInitializer;
+import io.trino.plugin.hive.security.SqlStandardAccessControlMetadata;
+import io.trino.spi.connector.ConnectorPageSourceProvider;
+import io.trino.spi.connector.ConnectorSplitManager;
+import io.trino.spi.type.TestingTypeManager;
+import org.apache.hadoop.fs.Path;
+
+import java.util.Optional;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.util.concurrent.MoreExecutors.newDirectExecutorService;
+import static io.airlift.concurrent.Threads.daemonThreadsNamed;
+import static io.trino.plugin.hive.HiveTestUtils.getDefaultHivePageSourceFactories;
+import static io.trino.plugin.hive.HiveTestUtils.getDefaultHiveRecordCursorProviders;
+import static io.trino.plugin.hive.TestingThriftHiveMetastoreBuilder.testingThriftHiveMetastoreBuilder;
+import static io.trino.spi.connector.MetadataProvider.NOOP_METADATA_PROVIDER;
+import static io.trino.type.InternalTypeManager.TESTING_TYPE_MANAGER;
+import static java.lang.String.format;
+import static java.util.concurrent.Executors.newCachedThreadPool;
+import static java.util.concurrent.Executors.newScheduledThreadPool;
+import static org.testng.util.Strings.isNullOrEmpty;
+
+public class S3SelectDefaultTestConfig
```

Comment:
Maybe something like `S3SelectTestHelper`? `Config` is usually associated with config classes